libs/blocks/section-metadata/section-metadata.js

```diff
@@ -39,15 +38,19 @@ async function handleStickySection(sticky, section) {
   const main = document.querySelector('main');
   switch (sticky) {
     case 'sticky-top':
+    {
       const { debounce } = await import('../../utils/action.js');
       window.addEventListener('resize', debounce(() => handleTopHeight(section)));
       main.prepend(section);
       break;
+    }
     case 'sticky-bottom':
+    {
```

Comment:
Nit: `case 'sticky-top':` needed the scoping due to declaring the constant. `case 'sticky-bottom':` and `default` shouldn't need the parenthesis.

Reply:
True, good call. I waffled on consistency vs. need. This is updated w/ your recommend. Thanks